README.md

```diff
@@ -56,10 +58,6 @@ We welcome and encourage contributions to this library, especially new dataset l
 
 ### Citing
 
+Please, refer to [CITATION.cff](./CITATION.cff). Read more about CITATION files [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#about-citation-files).
 
-```
-TBA
-```
-
-
-When working with datasets, please cite the version of `soundata` that you are using **AND** include the reference of the dataset, which can be found in the respective dataset loader using the `cite()` method. 
+When working with datasets, please also cite the version of `soundata` that you are using **AND** include the reference of the dataset, which can be found in the respective dataset loader using the `cite()` method. 
```

Comment:
Maybe we could ask them to cite the paper instad of the _version of soundata they are using_?

Reply:
hmm I guess it is important that people says which version of Soundata they have run. They could cite the Zenodo entry, for instance but I guess we can reword it to something like:

> When working with datasets, please indicate the version of `soundata` that you are using **AND** include the reference of the dataset, which can be found in the respective dataset loader using the `cite()` method.

Since we are already telling everyone to cite the paper by redirecting them to CITATION.cff file. Is this better?